### DISCUSSION TITLE
Delta ISP (The Netherlands)

### DISCUSSION THREAD

**Discussion by lcdvt** [fabianishere/udm-iptv#307] [Q&A] [answered]
Recently I switch to Delta from Caiway to force switching to XGS-PON and IPv6. Delta and Caiway are the same fiber network (Delta Fiber), but I could not get Caiway to switch me to XGS-PON.

Caiway and Delta use VLAN 100 for internet and VLAN 101 for IPTV. When I was still with Caiway I had a Genesis Fiber modem with Internet on UTP port 1 and IPTV on UTP port 2. However I've always used only UTP port 1 to my router, without any problem because IPTV also works on VLAN 100. For a long time I was using the USG from Unifi without any problems. Since last November I switched to the UniFi UXG-Lite also without any problems. Then in January I updated the UXG to v3.1.16 and started having the Amino IPTV receivers freeze the image for a few seconds several times a night. Often I had to switch channels to get it going again. I started an issue with UniFi but no progress. Then I switched from Caiway to Delta and directly replaced the default Nokia Fiber modem for my own ONT, the Huawei OptiXstar EN8010Ts-20. Al was working well except for IPTV, still the sam issues.
The problem with the UXG is that you can only tag 1 VLAN on the WAN port. Trying to solve this I used a UniFi Switch Lite between my ONT and the UXG to split the VLAN's which works great for IPTV however with an annoying side effect, now the UniFi Network discovers all devices behind the ONT.  So after a while I see about 180 'Cig Shanghai Co Ltd' devices in the UniFi GUI, which I believe are devices in a fiber distribution hub in our neighborhood, which is screwing up my UniFi Topology view. I can only solve this by blocking them in my device list in the UniFi GUI. Not a really nice solution.

Long story short, I thought I would try udm-iptv
I've played around with udm-iptv for a while now, but I'm unable to get it to work. Probably because this is a little bit out of my comfort zone. 

I found a technical document: [https://www.delta.nl/media/12gp0gk4/deltafiber_vrijemodemkeuze_pon-v1-2.pdf](https://www.delta.nl/media/12gp0gk4/deltafiber_vrijemodemkeuze_pon-v1-2.pdf ) from sep 2023 by Delta Fiber, describing the requirements.

VLAN's:
<img width="672" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/81c7d5ad-0889-43ae-a6d0-c5c2fbda917b">

Routing:
<img width="863" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/939e3183-8f6b-45e3-a295-281d54f043a5">

So I tried this with the following configuration result:

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth1
WAN VLAN: 101 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 62.45.57.34/32 62.45.57.0/24 62.45.59.0/24 62.45.76.0/24 62.45.58.226/32 62.45.45.150/32 62.45.49.0/24 212.115.196.0/25 217.63.90.128/25 217.63.91.0/26 62.45.61.32/27 62.45.61.16/28 62.45.61.64/28 217.102.255.57/32
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
17: iptv@eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.191.69.173/21 brd 10.191.71.255 scope global iptv
       valid_lft forever preferred_lft forever
default via 10.191.64.1 metric 217 
10.191.64.0/21 proto kernel scope link src 10.191.69.173 
=== Service Logs ===
Feb 15 21:45:04 UXG-Lite udm-iptvd[604220]: Obtaining IP address for VLAN interface
Feb 15 21:45:04 UXG-Lite udm-iptvd[604257]: udhcpc: started, v1.30.1
Feb 15 21:45:04 UXG-Lite udm-iptvd[604257]: udhcpc: sending discover
Feb 15 21:45:04 UXG-Lite udm-iptvd[604257]: udhcpc: sending select for 10.191.69.173
Feb 15 21:45:04 UXG-Lite udm-iptvd[604257]: udhcpc: lease of 10.191.69.173 obtained, lease time 1800
Feb 15 21:45:04 UXG-Lite udm-iptvd[604220]: NATing IPTV network ranges (if necessary)
Feb 15 21:45:04 UXG-Lite udm-iptvd[604220]: Creating static routes (if necessary)
Feb 15 21:45:04 UXG-Lite udm-iptvd[604220]: Setting up IGMP Proxy
Feb 15 21:45:04 UXG-Lite udm-iptvd[604220]: Using improxy...
Feb 15 21:45:04 UXG-Lite udm-iptvd[604220]: Starting IGMP Proxy

When I start one of my Amino IPTV receivers is starts up does som network initialization, but at the final step it says: "Checking connection to ACS" which fails. I believe this has something to do with authentication.
As I said this is all a bit outside of my comfort zone, may this be associated with:
<img width="574" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/7df3e822-7b5e-4985-81d1-f84d0621802b">
I do not know how to set that.

Besides it not working I also get other problems where many websites are no longer reachable from my network, lets say 'ping github.com' will fail, but google.com is still reachable, while on the other hand I can 'ping github.com' from the console of the UXG. To recover from this state I have to uninstall idm-iptv and reboot the UXG.

Any advise will be most welcome.

**Comment by lcdvt** [comment, marked as answer]
Never mind. Not using this project anymore. I'm using a Gateway Ultra now and don't need a 2.5Gb wan port 1Gb is enough so I'm just using a patch cable.
ISP on port 3 to split into vlan's
Internet vlan to port 4
Patch port 4 to wan port
Default, IPTV and other vlan's except internet to the rest of my network switches. 

Works like a charm with the standard unifi software

<img width="885" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/2c6e0c1f-32f2-491f-8bcc-330f57630f9f">
<hr>
<img width="758" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/3948b8e8-cbb6-4ff1-bbe3-7262e518b89c">
<hr>
<img width="755" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/18f1eae9-176e-4913-877d-bd419d2bde71">
<hr>
<img width="757" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/1176fe44-8582-4a13-9ba5-b86c9d40a97a">
<hr>
<img width="1281" alt="image" src="https://github.com/fabianishere/udm-iptv/assets/1609531/9fbb2a0c-0d8c-4b5d-8723-058d31477d98">

**Reply by ROosterloo1988** [reply]
> Never mind. Not using this project anymore. I'm using a Gateway Ultra now and don't need a 2.5Gb wan port 1Gb is enough so I'm just using a patch cable. ISP on port 3 to split into vlan's Internet vlan to port 4 Patch port 4 to wan port Default, IPTV and other vlan's except internet to the rest of my network switches.
> 
> Works like a charm with the standard unifi software
> 
> 

Can you make your patches visable? Because i want to implement it with an uxg-lite at my parents.
